### DISCUSSION TITLE
Support for Python 3.12

### DISCUSSION THREAD

**Discussion by joshTax** [serverless/serverless#12353] [Ideas]
Currently when specifying the runtime as `Python3.12` I will see this error from serverless:
```
Configuration error at 'provider.runtime': must be equal to one of the allowed values [dotnet6, go1.x, java17, java11, java8, java8.al2, nodejs14.x, nodejs16.x, nodejs18.x, provided, provided.al2, python3.7, python3.8, python3.9, python3.10, python3.11, ruby2.7, ruby3.2]
```

Python3.12 is now supported by AWS Lambda: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

**Comment by FinchPowers** [comment]
There is already a PR opened for it: https://github.com/serverless/serverless/pull/12300
But looking at the repo activity, I wonder if they stopped maintaining it now that they are working on V4.

https://www.serverless.com/blog/serverless-framework-v4-a-new-model
